config/routes.rb

```diff
@@ -254,7 +254,7 @@
     resource :no_access, only: :show, controller: "no_access"
     resource :start_registrations, path: "/start-registration", only: :show
 
-    authenticated :user, ->(user) { FeatureFlag.active?(:participant_validation, for: user.teacher_profile&.participant_profiles&.ecf&.active_record&.first&.school) } do
+    authenticated :user do
```

Comment:
Think this can go as it was only needed to get the context for the feature flag check

Reply:
Ah nice. Also, we have a multistep form for this kind of thing, I'll try to change it later on. :)